common/settings/settings.ts

```diff
@@ -39,6 +39,7 @@ export interface AnkiSettings {
     readonly maxImageHeight: number;
     readonly surroundingSubtitlesCountRadius: number;
     readonly surroundingSubtitlesTimeRadius: number;
+    readonly clickToMineDefaultPlayback: PostMinePlayback;
```

Comment:
Looking at this now the settings categorization does seem somewhat arbitrary but I think I would still prefer for this setting to be under `MiscSettings`.